sdk/containerservice/mgmt/src/main/java/com/azure/management/containerservice/KubernetesCluster.java

```diff
@@ -40,6 +42,12 @@
     /** @return the Kubernetes configuration file content with user-level privileges to the cluster */
     byte[] userKubeConfigContent();
 
+    /** @return the Kubernetes credentials with administrative privileges to the cluster */
+    List<CredentialResult> listadminKubeConfigs();
```

Comment:
Just `adminKubeConfigs`?

Reply:
Is it confused with above one `adminKubeConfigContent`?

Reply:
Better to have a name which matches the functionality. User your expertise :-)

Personally I prefer complete name like `adminKubernetesConfigures` or `adminKubernetesConfigureCredentials` (return type is `CredentialResult`, so it is config or credential?)

Reply:
Maybe `KubeConfig` is better than `KubernetesConfigurate` since the default file name is `kube.config`. So I think the final name could be `adminKubeConfigCredentials`?

Reply:
I am still not sure whether it is credential or config...

If it is credential, this name should be fine. If it is config, maybe we can name it `adminKubeConfigList`?

```
CredentialResult

    /**
     * The name of the credential.
     */
    @JsonProperty(value = "name", access = JsonProperty.Access.WRITE_ONLY)
    private String name;

    /**
     * Base64-encoded Kubernetes configuration file.
     */
    @JsonProperty(value = "value", access = JsonProperty.Access.WRITE_ONLY)
    private byte[] value;
```

Reply:
In kubernetes, the cluster credential is stored in a config file named `kube.config`, which will be read by `kubectl`. This is what the `value` is. I'm not sure it is said as credential or config.

Reply:
Finally, I think naming to `KubeConfigs` is better.